tools/static-assets/skel-full/imports/api/links/links.js

```diff
@@ -2,4 +2,6 @@
 
 import { Mongo } from 'meteor/mongo';
 
-export const Links = new Mongo.Collection('links');
+const Links = new Mongo.Collection('links');
+
+export { Links };
```

Comment:
My thinking is:
1. It indicates that it will be likely that you'll export more from this file (methods, etc)
2. I think it's a good convention to place all exports at the base of each file, otherwise it can be difficult to determine what's exported